src/test/java/org/elasticsearch/cloud/aws/AmazonS3Wrapper.java

```diff
@@ -88,6 +93,21 @@ public ObjectListing listObjects(ListObjectsRequest listObjectsRequest) throws A
     }
 
     @Override
+    public ListObjectsV2Result listObjectsV2(String s) throws SdkClientException, AmazonServiceException {
```

Comment:
nvm maybe.. github hides old methods too well. But isn't it weird that the wrapper exposes like everything?